.github/workflows/maven.yml

```diff
@@ -13,16 +13,16 @@ jobs:
     name: Build and Unit Tests
     runs-on: ubuntu-latest
     env:
-      JAVA_DOWNLOAD_CHECKSUM: 2226366d7dffb3eb4ec3d14101f4ddb4259195aa43bb319a93810447b8384930
+      JAVA_DOWNLOAD_CHECKSUM: 6c06853332585ab58834d9e8a02774b388e6e062ef6c4084b4f058c67f2e81b5
     steps:
       - uses: actions/checkout@v1
       - run: |
-          wget "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.1%2B9/OpenJDK13U-jdk_x64_linux_hotspot_13.0.1_9.tar.gz"
-          echo "${JAVA_DOWNLOAD_CHECKSUM}  OpenJDK13U-jdk_x64_linux_hotspot_13.0.1_9.tar.gz"  | sha256sum -c
-      - name: Set up JDK 13
+          wget "https://github.com/AdoptOpenJDK/openjdk14-binaries/releases/download/jdk-14%2B36/OpenJDK14U-jdk_x64_linux_hotspot_14_36.tar.gz"
```

Comment:
Looks like AdoptOpenJDK released an [Action](https://github.com/AdoptOpenJDK/install-jdk) so probably no reason to hack the `actions/setup-java` action.
I guess it supports Java 14 since it's calling the APIs.

Reply:
yeap, it worked. thanks :)